diff --git a/ios/DevMenuViewController.swift b/ios/DevMenuViewController.swift
index 2469f1ae21826abba2a4445c906b2d40a6a4cc29..9fdc3ca871514311dddaf2af38705586a121514e 100644
--- a/ios/DevMenuViewController.swift
+++ b/ios/DevMenuViewController.swift
@@ -63,7 +63,11 @@ class DevMenuViewController: UIViewController {
   // MARK: private
 
   private func initialProps() -> [String: Any] {
-    let isSimulator = TARGET_IPHONE_SIMULATOR > 0
+#if targetEnvironment(simulator)
+    let isSimulator = true
+#else
+    let isSimulator = false
+#endif
 
     return [
       "showOnboardingView": manager.shouldShowOnboarding(),
